### PR TITLE
Add additional base test suite tests

### PR DESCRIPTION
**Pull request by loeakaodas**
# Description of change
added missing tests as per [TDL-14049](https://jira.talendforge.org/browse/TDL-14049)

# Manual QA steps
 - ran tap-tester locally
 
# Risks
 - 
 
# Rollback steps
 - revert this branch

### CODE REVIEW

File: tests/test_github_automatic_fields.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    def name():
+        return "tap_tester_github_automatic_fields"
+
+    def test_run(self):
+        """
+        Verify that for each stream you can get multiple pages of data
+        when no fields are selected and only the automatic fields are replicated.
+        """
+
+        expected_streams = set([
```
Comment (kspeer825) [inline tests/test_github_automatic_fields.py:22]: This should run off of the expected streams defined in base.py

File: tests/test_github_automatic_fields.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+    @staticmethod
+    def name():
+        return "tap_tester_github_automatic_fields"
+
+    def test_run(self):
+        """
+        Verify that for each stream you can get multiple pages of data
+        when no fields are selected and only the automatic fields are replicated.
+        """
```
Comment (kspeer825) [inline tests/test_github_automatic_fields.py:20]: This assertion is missing`Verify that all replicated records have unique primary key values`.  See here for test standards https://github.com/stitchdata/tap-tester/blob/master/reference/expectations/saas_taps.md#assertions-1

**Review by kspeer825 — CHANGES_REQUESTED** [review]
Left a few comments

File: tests/test_github_automatic_fields.py (hunk 1 above)
Comment (loeakaodas) [reply tests/test_github_automatic_fields.py]: bug added to Jira and indicated in the code.

File: tests/base.py [hunk 3]
```diff
@@ -83,7 +83,7 @@ def expected_check_streams():
             'reviews',
             'stargazers',
             'team_members',
-            'team_memberships',
+            # 'team_memberships',
```
[the remark this answers is not in the document]
Comment (loeakaodas) [reply tests/base.py]: updated to use `expected_streams` method and exclude streams that are not testable.

**Review by kspeer825 — CHANGES_REQUESTED** [review]
This is missing the proper .circleci/config.yml so that tests will run on schedule as regression protection. In addition the bug ticket linked in the test does not include any details of the failure or how to reproduce the issue.

**Comment by loeakaodas** [comment]
@kspeer825 I wasn't sure what you meant by the circle config running on a schedule, but I adjusted the `config.yml` file to the latest image. I believe that it was already setup to run daily using a cron schedule.

Also added additional detail to the bug ticket.

**Review by kspeer825 — COMMENTED** [review]
Is this PR still valid, or can it be closed?

**Comment by loeakaodas** [comment]
Closing this PR, re-submitted in #145

**Pull request closed without merging**